CONTRIBUTING.md

```diff
@@ -0,0 +1,12 @@
+# What we are looking for
+
+Thank you for your interest in contributing! We are looking for additional podcasts to add to our expanding list.
+
+To contribute, please open a PR with the name of the podcast, a link, and which category you think it would fit best.
```

Comment:
Where should the users put the podcast/link/category?

Should they update the readme and open the PR with those changes?
Should they fork the repository to be able to create a PR if they are not contributors?